lib/jekyll/commands/compose.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+module Jekyll
+  module Commands
+    class ComposeCommand < Command
+      def self.init_with_program(prog)
+        prog.command(:compose) do |c|
+          c.syntax "compose [--post|--draft|--collection COLLECTION] NAME"
```

Comment:
Was this there in the previous implementation?
Either ways, please remove the *options* from the syntax. That is going to confuse users.

```suggestion
          c.syntax "compose NAME"
```
This should default to using `--post` if no options were provided.